prototool.yaml

```diff
@@ -20,6 +21,44 @@ lint:
       - ENUM_NAMES_CAPITALIZED
     # remove:
     #   - ENUM_NAMES_CAMEL_CASE
+  ignores:
+    - id: ENUM_FIELD_NAMES_UPPER_SNAKE_CASE
+      files:
+        - protos/exchange/exchange_message.proto
+    - id: ENUM_NAMES_CAMEL_CASE
+      files:
+        - protos/exchange/exchange_message.proto
+    - id: ENUM_NAMES_CAPITALIZED
+      files:
+        - protos/exchange/exchange_message.proto
+    - id: MESSAGE_FIELD_NAMES_LOWER_SNAKE_CASE
+      files:
+        - protos/exchange/exchange_message.proto
+    - id: ENUM_FIELD_NAMES_UPPER_SNAKE_CASE
+      files:
+      - protos/exchange/exchange_message.proto
+      - protos/protocol/core/Tron.proto
+    - id: ENUM_NAMES_CAMEL_CASE
+      files:
+      - protos/exchange/exchange_message.proto
+      - protos/protocol/api/api.proto
+    - id: ENUM_NAMES_CAPITALIZED
+      files:
+      - protos/exchange/exchange_message.proto
+      - protos/protocol/api/api.proto
+      - protos/protocol/core/Tron.proto
+    - id: MESSAGE_FIELD_NAMES_LOWER_SNAKE_CASE
+      files:
+      - protos/exchange/exchange_message.proto
+      - protos/protocol/api/api.proto
+      - protos/protocol/core/Discover.proto
+      - protos/protocol/core/Tron.proto
+    - id: MESSAGE_NAMES_CAPITALIZED
+      files:
+      - protos/protocol/core/Tron.proto
+    - id: RPC_NAMES_CAPITALIZED
+      files:
+      - protos/protocol/api/api.proto
```

Comment:
Added a ticket to handle this gracefully in the future: https://tronnetwork.atlassian.net/browse/BTFS-1569